tsconfig.json

```
@@ -3,6 +3,9 @@
   "compilerOptions": {
     "baseUrl": "./",
     "paths": {
+      "@defichain-app-lib/*": [
+        "apps/libs/*/src"
+      ],
```

```suggestion
      "@defichain-apps/libs-*": [
        "apps/libs/*/src"
      ],
```

Probably better to structure the import like this? To keep everything contextually related within the same org prefix.

Also; we are using organization prefix and `@defichain-app-lib` is not reserved while [`@defichain-apps`](https://www.npmjs.com/settings/defichain-apps/packages) is.

---

Great idea thanks